manifests/params.pp

```diff
@@ -45,7 +45,7 @@
   #     can be set to the user field that is in the group group_field
   $oned_ldap_user_group_field = hiera('one::oned::ldap_user_group_field','undef'),
   # should we enable opennebula repos?
-  $one_repo_enable = hiera('one::enable_opennebula_repo', 'true' ),
+  $one_repo_enable = hiera('one::enable_opennebula_repo', true ),
```

Comment:
OK. That is valid.
But that also means that we should take care on the two possible values in prerequisites.pp:
- 'true' (if set in hiera)
- true (if not set in hiera)